### DISCUSSION TITLE
CopyQ synchronization

### DISCUSSION THREAD

**Discussion by GeosDroid** [hluk/CopyQ#2420] [General]
I have a failure to synchronize. I want to sync CopyQ on my Win10 desktop & laptop. Starting with the desktop I followed all the steps and selected a Dropbox folder for the tabs I want to sync. I have no idea what to do on the laptop. In Preferences I don't see how to sync. I don't see anything in the Dropbox folder.

**Comment by eggbean** [comment]
If you open the CopyQ windows and type ctrl-p to get Preferences, then to the "Items" section in the left-hand pane, and the "Synchonize" in the middle pane, you can put in the directory locations for the tabs. Remember to include any `&` character in the tab names (which precede the letter used to select the tab after pressing the alt key). You can get this "true" name of the tab by right clicking the tab and selecting "Rename...".
